recipes/vtu11/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.8)
+
+project(test_package CXX)
+
+find_package(vtu11 REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE vtu11::vtu11)
```

```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE vtu11::vtu11)
target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```
Consumer should set which C++ standard is wanted/supported. 
In case it fail due the compiler version vs standard required, we can add a if..else here in cmake to set the standard according to the compiler version too.

---

This should for add a second target for C++17 testing